tests/executables_src/testDevice.cpp

```diff
@@ -176,13 +167,6 @@ BOOST_AUTO_TEST_CASE(testIsFunctional) {
 
   d.close();
   BOOST_CHECK(d.isFunctional() == false);
-
-  ChimeraTK::BackendFactory::getInstance().registerBackendType("DysfunctDummy", &DysfunctDummy::createInstance);
```

Comment:
Removing this tests is good because I think open() on the dysfunct dummy should have thrown. I guess that the device is still open after throwing an exception is part of the unified backend test. To test it here make no sense because it is implementation specific.